lib/cadet/course/course.ex

```diff
@@ -57,6 +56,30 @@ defmodule Cadet.Course do
     end
   end
 
+  @doc """
+  Create group entity with given name
+  """
+  def create_group(name, leader, mentor) do
+    changeset =
+      %Group{}
+      |> Group.changeset(%{name: name})
+      |> put_assoc(:leader, leader)
```

Comment:
We agreed not to use `put_assoc` anymore. I'm assuming a group must have both a leader and mentor. If so, then refer to #82 (can also refer to #144 where this was implemented for the `Assessments` context)

Reply:
On reading the issues and codebase, should this be changed to:
|> change(%{"leader_id": Map.get(leader, :id)})
|> change(%{"mentor_id": Map.get(mentor, :id)})